easybuild/easyconfigs/p/PyTorch/PyTorch-1.6.0-fosscuda-2019b-Python-3.7.4.eb

```
@@ -0,0 +1,312 @@
+    '90f20492621d5ed80b442aa682ff92d7ccf333ac8fac4a10e7e02afb159f3c13',  # FP16-20200514.tar.gz
+    '9ccf554541666b5c089ad5dd465141d671c99971f36d72f313652f5c49ffce14',  # FXdiv-20200417.tar.gz
+    '1f11dbbfad78d0a4c39fe94e52a28c0821cb25f9880420bb304f6302f73fe002',  # NNPACK-20191007.tar.gz
+    '0d752bd75f46ce4d7c6f0a60b0d6c0e5918a7b4683c825284f8db3706dd24f76',  # QNNPACK-20190828.tar.gz
+    'b55a6ef3a0b4c0d3c39ad5578eb4fa9a7b2d7ee1ef06592b8a808a59a8e6589b',  # XNNPACK-20200323.tar.gz
+    '0de43b6eaddd356f1d6cd164f73f37faf2f6c96fd684e1f7ea543ce49c1d144e',  # benchmark-20180606.tar.gz
+    '18a99130ced1eaacab2ba8f75a1435f9955aab54fa0436b60468f020876ee902',  # cpuinfo-20200611.tar.gz
+    '7224b03af4acbc54525105bb42e3ecd75c66a3a5b47e8a725ab008467c4109f9',  # cub-20170829.tar.gz
+    '2ec954f18cec50a7063a7358ce555f7e11788a7f6d4e7e597d83687dc2f3b989',  # eigen-20190125.tar.gz
+    'c2e122ea2c3a571207ef0ba9c495444bda4b83927d076ed04fbb6383ae428441',  # fbgemm-20200531.tar.gz
+    '5c7ecb03ede09a38fa9bcf2527a0f1454cd67a34c0e6fd939ae4d93b276ce982',  # asmjit-20200429.tar.gz
+    '3f2dc1970f397a0e59db72f9fca6ff144b216895c1d606f6c94a507c1e53a025',  # cpuinfo-20190201.tar.gz
+    'e99b904983d08ac8e9bddb5b0d21196b78ad9499e3c5d12192cee2ddd2b7515c',  # googletest-20180925.tar.gz
+    '92b1ba081c55aca384d975bb2069e0ecc5f71a35f894a2047d9783ef6800b41e',  # fmt-20200405.tar.gz
+    '982100f4a514b0678e3161e39c3660dc7be3c83bdc1e39617066dc49712d50ac',  # foxi-20191220.tar.gz
+    'fdd6f08bdb33d33f4df516ffb91730fdb163479c19502cfc983083fd9cf43bfa',  # gemmlowp-20181126.tar.gz
+    '5243b523dba4aa55090790c9836803c671daa58bff24411e52267e119f4a61e0',  # gloo-20200317.tar.gz
+    'd0d447b4feeedca837a0d46a289d4223089b32ac2f84545fa4982755cc8919be',  # googletest-20180831.tar.gz
+    'ab37b266702557b819a598d1bf89946cba281a8d49224ce105280eb657d0c733',  # ideep-20200616.tar.gz
+    '4bc7dafaaaa55db55bb2654e043cc7f3fb35cd49d19c4efee2dbce1ab863c179',  # mkl-dnn-20200605.tar.gz
+    'c3bf8cc8091b24c43aa205a1f2037a518d249eb5997bea7f543b7fc880e80e21',  # onnx-20200531.tar.gz
+    'c7682e9007ddfd94072647abab3e89ffd9084089460ae47d67060974467b58bf',  # benchmark-20180525.tar.gz
+    '314cde420a7cf692bdb6877bc6af6bc514805f6cdb8bee90f32566ed08d94b1c',  # onnx-tensorrt-20190916.tar.gz
+    '2435b7fb83b8a608c24ca677907aa9a35e482a7f018e65ca69481b3c8c9f7caf',  # protobuf-20200214.tar.gz
+    'f6c4dab91ae9a03b3019e7cab0572743afd0e1b6e75b97fcca50259c737c924e',  # psimd-20200517.tar.gz
+    'd13818a10a645d557202aa005606337b2a0cdf690e5655253800ab11abb0c076',  # pthreadpool-20200615.tar.gz
+    '1a1ccf8779332a7d6d0db9034d42df188867cbaf52eb71d74451f79aa8ce2040',  # pybind11-20190204.tar.gz
+    '13100c3deed300bbf16f87d8af3539f432462bfef9d38f0c7e3e387dc2e88676',  # PeachPy-20180218.tar.gz
+    '8cb5fae822077ca9cbc14dcc7bba9a3a35ad519284fc5169f9a176672c63860a',  # sleef-20190730.tar.gz
+    'be111cf161b587812fa3b106fe550efb6f129b8b0b702fef32fac23af9580e5e',  # tbb-20181009.tar.gz
```

I have an existing download from `https://github.com/intel/tbb/archive` named `tbb-20181009.tar.gz`, which has a different checksum...

The contents of the tarball are exactly the same though (no surprise there), so we could add 2 alternative checksums:

```suggestion
    # checksum for original download of tbb-20181009.tar.gz from https://github.com/intel/tbb/archive
    ('dc0a8d8d96cb8765782aa6ac1b509ad4db955d9bbb58fa5cc2265f0292756d72',
    # checksum for download of tbb-20181009.tar.gz from https://github.com/01org/tbb/archive
     'be111cf161b587812fa3b106fe550efb6f129b8b0b702fef32fac23af9580e5e'),
```

see also #10367